addon/services/scheduler.js

```diff
@@ -183,7 +191,7 @@ if (DEBUG) {
      */
     hasActiveQueue() {
       if (!this.queues) {
-        return true;
+        return false;
```

Comment:
Is this intended? Returning `false` will force this waiter to wait in perpetuity. Is that actually what we want?

Reply:
Which logic in particular is hard to understand? The reason I changed this from `true` to `false` is because the waiter was going to continue waiting even when there were no more active queues.

Reply:
Ah, you're correct. My previous comment was wrong. Makes sense!